Source/Parser/SVG/SVGParserPrimitives.swift

```diff
@@ -200,6 +200,11 @@ public class SVGHelper: NSObject {
                let height = SVGLengthParser.yAxis.double(string: nums[3], context: context) {
                 return CGRect(x: x, y: y, width: width, height: height)
             }
+        } else if let widthAttr = attributes[ignoreCase: "width"],
+                  let heightAttr = attributes[ignoreCase: "height"],
+                  let width = SVGLengthParser.xAxis.double(string: widthAttr, context: context),
+                  let height = SVGLengthParser.yAxis.double(string: heightAttr, context: context) {
```

Comment:
The `SVGLengthParser` parse `%` props to absolute values based on the parent width, height (fetched from context). 

I wonder what would be the values parsed here for width and height, can u please check?

Reply:
Refer to the results in viewport-02.ref, if the `%` is used in the viewport attribute, the size will be 100x100.